src/Service/Search/SolrSearch.php

```diff
@@ -29,6 +29,20 @@
  */
 class SolrSearch implements Search
 {
+    public const QUERY_FIELDS_REQUIRED = [
+        'url',
+        'title',
+        'description',
+        'id',
+        'sp_id',
+        'sp_objecttype',
+        'sp_date',
+        'sp_date_list',
+        'sp_date_from',
+        'sp_date_to',
+        'sp_meta_*',
+    ];
```

Comment:
Yes, that's not ideal. Unfortunately, the information about which fields are required is also not known. I think you can do without `sp_date_from` and `sp_date_to`. They are no longer filled with the new indexer anyway. It should also be possible to determine them via `date_list`. We'll have to leave the others as they are for the time being. So far I can't think of a solution.

Reply:
Alright, I removed `sp_date_from` and `sp_date_to`